src/design-system/components/dialog/DialogTitle.tsx

```diff
@@ -7,10 +7,11 @@ export type DialogTitleProps = {
   text?: string;
   subtext?: string;
   onCloseClick?: () => void;
+  onCloseDataTestId?: string;
```

Comment:
```suggestion
  closeDataTestId?: string;
```

The `on` prefix suggests that its a callback